crates/bevy_ecs/src/system/query.rs

```diff
@@ -1125,6 +1125,7 @@ where
     ///
     /// ```
     /// # use bevy_ecs::prelude::*;
+    /// # use bevy_ecs::
```

Comment:
```suggestion
```

Reply:
My question is how did this get past the 2 prior reviewers?

Reply:
I did give the changes a proper look, but clearly I need to be more focused when reviewing. My apologies.

Reply:
Yeah, no worries. Wasn't meant to be an attack, just a joking comment.

This kind of stuff is why we have CI.